node-module-system/calculate.js

```diff
@@ -0,0 +1,29 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import divide from './divide.js';
+import multiply from './multiply.js';
+
+if (process.argv[3] === 'plus') {
+  console.log('result:', add(Number(process.argv[2]), Number(process.argv[4])));
```

Comment:
You convert to Number over and over again. It would be nice to do it once in one place.